Assets/Mirror/Core/NetworkServer.cs

```diff
@@ -315,7 +315,19 @@ static void OnCommandMessage(NetworkConnectionToClient conn, CommandMessage msg,
                 // Ignore commands that may have been in flight before client received NotReadyMessage message.
                 // Unreliable messages may be out of order, so don't spam warnings for those.
                 if (channelId == Channels.Reliable)
+                {
+                    // Attempt to identify the target object, component, and method to narrow down the cause of the error.
+                    if (spawned.TryGetValue(msg.netId, out NetworkIdentity netIdentity))
+                        if (netIdentity.NetworkBehaviours.Length > msg.componentIndex && netIdentity.NetworkBehaviours[msg.componentIndex] is NetworkBehaviour component)
+                            if (RemoteProcedureCalls.GetInvokerForHash(msg.functionHash, RemoteCallType.Command, out Invoker invoker))
```

Comment:
this is useful. could we make this a helper: RemoteProcedureCalls.GetFunctionNameFromHash(int hash)? and then reuse it below as well?

Reply:
I thought about that but wasn;t sure if you'd want that, or where exactly you'd want it.